InsertionSort.java

```diff
@@ -0,0 +1,27 @@
+public class MyInsertionSort {
+ 
+    public static void main(String a[]){
```

Comment:
You can get rid of this method.
We use the main method provide in the App class of this project as a starting point.